core/src/main/java/org/elasticsearch/bootstrap/SystemCallFilter.java

```diff
@@ -243,6 +243,8 @@ static SockFilter BPF_JUMP(int code, int k, int jt, int jf) {
         Map<String,Arch> m = new HashMap<>();
         m.put("amd64", new Arch(0xC000003E, 0x3FFFFFFF, 57, 58, 59, 322, 317));
         m.put("i386",  new Arch(0x40000003, 0xFFFFFFFF, 2, 190, 11, 358, 354));
+        // 64-bit only!
+        m.put("aarch64",  new Arch(0xC00000B7, 0xFFFFFFFF, 1079, 1071, 221, 281, 277));
```

Comment:
Maybe change the comment to "arm is 64 bit only" so it doesn't get confused with the line above it supporting 32 bit intel.